mlflow/ml-package-versions.yml

```diff
@@ -359,7 +359,7 @@ prophet:
     pip_release: "prophet"
 
   models:
-    minimum: "1.1.1"
```

Comment:
we need to think about how to handle dead packages.

Reply:
we can fully drop prophet support, if it is dead.

Reply:
or we can leave support for its last version.

Reply:
Supporting the last version sounds good.